packages/community-cli-plugin/src/commands/start/attachKeyHandlers.js

```diff
@@ -120,3 +145,11 @@ export default function attachKeyHandlers({
     ].join('\n'),
   );
 }
+
+function setRawMode(enable: boolean) {
+  invariant(
```

Comment:
You're certain you want to make this an invariant instead of just _not_ setting raw mode if it's unavailable?

Reply:
Yup. This is gated earlier by `process.stdin.isTTY` and should never throw — it's an invariant for type refinement. Matches Meta-internal version.